### DISCUSSION TITLE
How to make private sandbox public while forking?

### DISCUSSION THREAD

**Discussion by jcubic** [codesandbox/codesandbox-client#6858] [Q&A] [answered]
I'm following the ReactJS course when one of the examples is private, I can watch it but I can't fork or save it because I don't have a pro account. I get this error:

```
We were unable to fork the sandbox
You need to be a patron to create a private or unlisted sandbox
```

Is there a way to make the sandbox public while forking?

**Comment by JamesACS** [comment, marked as answer]
Hey @jcubic - You would need to contact the owners of the Sandbox to change the permissions to public, or upgrade your account to Pro
